src/routes/[...dynamicRoute]/+page.server.ts

```diff
@@ -23,18 +20,19 @@ const query = gql`
   }
 `;
 
-export const load = (async ({ params }): Promise<DraftNavigationLink> => {
+export const load = (async ({
+  params,
+  locals: { contentfulClient },
+}): Promise<{ text: string }> => {
+  // TODO: create fallback fixture
```

Comment:
just highlighting this in case we want to specifically capture this in a follow-up story